crates/client/eth-client/src/config.rs

```diff
@@ -70,6 +74,90 @@ pub struct HttpProviderConfig {
     pub gas_price_poll_ms: Option<u64>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(tag = "oracle_name", content = "config")]
+pub enum OracleConfig {
```

Comment:
Can you put everything pragma related in a module/file

Reply:
putted it in an oracle.rs but i was wondering if its good enough or should i put it in a separate folder

Reply:
It is good enough for now.
In the long run you will create a dedicated sdk crate